Compilation fails

Trying to compile the latest master locally fails

```
$ cargo --version
cargo 1.42.0 (86334295e 2020-01-31)
```

```
$ cargo test
   Compiling jpeg-decoder v0.1.18 (/Users/olojkine/Developpement/jpeg-decoder)
error[E0432]: unresolved import `png::HasParameters`
 --> examples/decode.rs:4:5
  |
4 | use png::HasParameters;
  |     ^^^^^^^^^^^^^^^^^^ no `HasParameters` in the root

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `jpeg-decoder`.
```

--- a/examples/decode.rs
+++ b/examples/decode.rs
@@ -1,7 +1,6 @@
 extern crate jpeg_decoder as jpeg;
 extern crate png;
 
-use png::HasParameters;
 use std::env;
 use std::fs::File;
 use std::io::{self, BufReader, Write};

